README.md

```diff
@@ -67,17 +68,15 @@ Every pull request must pass the following:
 2) Unit tests of each modified module must pass
 3) Integration tests must pass. to run the integration tests build the "deployer" module using maven
 
-
 ## Licensing
 **{code} does not provide legal guidance on which open source license should be used in projects. We do expect that all
 projects and contributions will have a valid open source license within the repo/project, or align to the appropriate 
 license for the project/contribution.** 
 The default license used for {code} Projects is the [MIT License](http://codedellemc.com/sampledocs/LICENSE "LICENSE").
 
-Ocopea kubernetes extension is freely distributed under the 
+Ocopea Kubernetes extension is freely distributed under the 
 [MIT License](http://emccode.github.io/sampledocs/LICENSE "LICENSE"). See LICENSE for details.
```

Comment:
Need to remove this section as we have our own license.

Reply:
Same is true for Support section, we should remove it.

Reply:
Would like to defer the change to License/Contributing until we have a consistent statement to apply across all projects.  The current PR only addresses minor changes.

Reply:
I added issue to track this. Not sure what you have in mind though. Removal of the sections today makes it consistent with all projects.